helix-term/src/application.rs

```diff
@@ -553,6 +559,15 @@ impl Application {
         }
     }
 
+    pub fn handle_virtual_terminal_events(&mut self, event: TerminalEvent) {
+        match event {
+            TerminalEvent::TitleChange(_, _) => (),
+            TerminalEvent::Update(_) => {}
+        }
+
+        self.render();
+    }
```

Comment:
Similar to `helix_dap`, can you move this to the `helix-vte` crate, then call it in the loop here (with a render after?) I'm going to extract the lsp handler into helix-lsp too